test/IntegrationTests/DotNetCliTests.cs

```diff
@@ -0,0 +1,131 @@
+// <copyright file="DotNetCliTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if !NETFRAMEWORK
+
+using System.Runtime.InteropServices;
+using FluentAssertions;
+using IntegrationTests.Helpers;
+using Org.BouncyCastle.Asn1.X509;
+using Xunit.Abstractions;
+using Timeout = IntegrationTests.Helpers.Timeout;
```

Comment:
Maybe it would be better to rename `IntegrationTests.Helpers.Timeout` class to `Timeouts`?

Reply:
This is a good idea, let's do a separate PR to handle that. For now I will use the `Timeouts` alias on this file.

Reply:
Following up via #2187